src/lib.rs

```diff
@@ -76,15 +76,23 @@ pub mod tokio_adapter;
 ///
 /// As of the time of this writing, the opentelemetry crate exposes no link information
 /// so this struct does not send link information.
-#[derive(Derivative)]
-#[derivative(Debug)]
 pub struct StackDriverExporter {
-  #[derivative(Debug = "ignore")]
   tx: futures::channel::mpsc::Sender<Vec<Arc<SpanData>>>,
   pending_count: Arc<AtomicUsize>,
   maximum_shutdown_duration: Duration,
 }
 
+impl fmt::Debug for StackDriverExporter {
+  fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
Super pedantic 🚲: this `fmt` trait method arg is called `f` [upstream](https://doc.rust-lang.org/std/fmt/trait.Debug.html) -- IMO there's usually not a reason to _not_ align with that. Thoughts?

Reply:
I see no reason not to align with, so I have -- I somehow thought it was usually called `fmt`, but I just found out that most `Debug` impls in Quinn (one larger codebase I'm familiar with) calls most of them `f`. Seemed like good fodder for [a Twitter poll](https://twitter.com/djco/status/1326508180357189633), too.